proposals/lip-0065.md

```diff
@@ -98,7 +98,7 @@ To summarize, we chose sparse Merkle trees because:
 
 | Name | Type | Value | Description |
 |------|------|-------|-------------|
-| `MAX_EVENT_SIZE_BYTES` | integer | `1024` | Maximum size in bytes of a serialized event. |
+| `MAX_EVENT_SIZE_BYTES` | integer | `16*1024` | Maximum size in bytes of a serialized event. |
```

Comment:
I have missed the background behind this change, but still fine from my side

Reply:
is to be able to emit complete ccms into events. I added a generous buffer (also 16 is a round number).